ios/FluentUI/List/ListItem.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+public typealias ListItemAccessoryType = TableViewCellAccessoryType
+public typealias ListItemBackgroundStyleType = TableViewCellBackgroundStyleType
+public typealias ListItemTokenSet = TableViewCellTokenSet
+
+/// View that represents an item in a List.
+public struct ListItem<LeadingView: View,
+                       TrailingView: View,
+                       Title: StringProtocol,
+                       Subtitle: StringProtocol,
+                       Footer: StringProtocol>: View {
+
+	// MARK: Initializer
+
+	/// Creates a ListItem view
+	/// - Parameters:
+	///   - title: Text that appears as the first line of text
+	///   - subtitle: Text that appears as the second line of text
+	///   - footer: Text that appears as the third line of text
+	///   - leadingView: The view that appears on the leading edge of the view
+	///   - trailingView: The view that appears on the trailing edge of the view, next to the accessory type if provided
+    public init(title: Title,
+                subtitle: Subtitle = String(""),
+                footer: Footer = String(""),
+                leadingView: @escaping () -> LeadingView = { EmptyView() },
```

Comment:
I'm not well versed in SwiftUI so apologies if this is a stupid question but why not just use optionals here instead of `EmptyView()`? (Especially considering SwiftUI documentation states "You will rarely, if ever, need to create an EmptyView directly" which sounds like this should be steered away from unless there's no other way).